common.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+option go_package = "tofnd;tofnd";
+
+package tofnd;
+
+// Key presence check types
+message KeyPresenceRequest {
+    string key_uid = 1;
+}
+
+message KeyPresenceResponse {
+    enum Response {
+	RESPONSE_UNSPECIFIED = 0;
```

Comment:
Indentation seems to be off?

Reply:
somehow spaces were converted to tabs -.- fixed